service/src/main/java/bio/terra/cbas/runsets/monitoring/SmartRunsPoller.java

```diff
@@ -186,6 +185,29 @@ private Run tryUpdateRun(Run r) {
     }
   }
 
+  /*
+  Method getWorkflowErrors is a reusable code from SmartRunsPoller that we'll move
+  with a workflow completion handling from a SmartRunsPoller.
+  Pending task [WM-2090].
+   */
+  public List<String> getWorkflowErrors(Run updatableRun) {
```

Comment:
Thanks to Janet, these should now be part of the callback (at least if triggered at callback time... If this is triggered at workflow status poll time, we'll still need to fetch them directly)

Reply:
Done